pkgs/development/python-modules/cfn-lint/default.nix

```diff
@@ -64,8 +58,8 @@ buildPythonPackage rec {
     "cfnlint.transform"
   ];
 
-  checkInputs = [ pytestCheckHook mock pydot ];
-  preCheck = "export PATH=$out/bin:$PATH";
+  # checkPhase collects 0 tests, then fails after running the empty test suite.
+  doCheck = false;
```

Comment:
```suggestion
  doCheck = false;
  pythonImportsCheck = [ "cfn-lint" ];
```

Reply:
No. `pythonImportsCheck` already exists in this file: https://github.com/NixOS/nixpkgs/blob/a2afcdc3192a4225be35af8b87c6329f36199c13/pkgs/development/python-modules/cfn-lint/default.nix#L45-L59